.github/release-initial-checklist.md

```diff
@@ -50,11 +50,11 @@ When creating testing notes, please write them from the perspective of a "user"
   * [ ] Confirm [WooCommerce.com plugin page](https://woocommerce.com/products/woocommerce-gutenberg-products-block/) is updated.
   * [ ] Download zip and smoke test.
   * [ ] Test updating plugin from previous version.
-* [ ] Merge this pull request back into `trunk`. This may have merge conflicts needing resolved if there are any cherry-picked commits in the release branch.
+* [ ] Merge this pull request back into `trunk`. This may have merge conflicts needing resolved if there are any cherry-picked commits in the release branch. After merging, GitHub will automatically delete the branch, restore it.
```

Comment:
Correct me if I'm wrong, but I think the intention is that `release/x.y` branches should not be deleted.

Reply:
Restoring them will cause the pull request automation to run again, so for now we just leave deleted - thus intentionally this doesn't mention restoring. They can be restored as needed for hotfix down the road (there's an open issue in the pull request automation to fix as well).

Reply:
Ah, that explains why there was a duplicate PR for 3.6.0 release (#3268). :smile: 

I reverted this change in 70a422bad5f4c33e29cd35bce8a2973e3bd35aa6.